### DISCUSSION TITLE
Is there completions available for usage 1.0.1 itself?

### DISCUSSION THREAD

**Discussion by powerman** [jdx/usage#146] [Q&A]
I've tried to setup it using
```sh
usage generate completion zsh usage >~/.zsh/completions/_usage
```
but it doesn't work (probably because `$spec` inside is not defined).

BTW, is it supposed to implement `usage --usage` or `usage completion usage`?

**Comment by jdx** [comment]
not yet, it's on my list of things to do

> BTW, is it supposed to implement usage --usage or usage completion usage?

yeah, that's what `mise g completion --usage-cmd` is for
